Dockerfile

```
@@ -19,8 +20,8 @@ USER ladybugbot
 WORKDIR ${WORKDIR}
 
 # Expects a decompressed radiance folder in the build context
-COPY radiance/usr/local/radiance/bin ${WORKDIR}/bin
-COPY radiance/usr/local/radiance/lib ${WORKDIR}/lib
+COPY radiance/usr/local/radiance/bin ${BINPATH}
+COPY radiance/usr/local/radiance/lib ${RAYPATH}
```

Sorry for making it not as good looking by adding the current directory to RAYPATH.

```suggestion
COPY radiance/usr/local/radiance/lib ${WORKDIR}/lib
```

---

It's ok :crying_cat_face: